Menu/Menu.py

```diff
@@ -116,29 +116,97 @@ def drawAllPlayersInLobby(self):
             pygame.draw.circle(self.screen, "green", (PLAYER_THREE_X_POS, CIRCLE_Y_POS), CIRCLE_RADIUS, CIRCLE_RADIUS)
             self.screen.blit(playerImage, (PLAYER_THREE_X_POS - AVATAR_PADDING, CIRCLE_Y_POS - AVATAR_PADDING))
 
-    def showLobby(self):
+    def showLobbyBackground(self):
         self.drawBackground()
         self.drawMenuText()
 
         self.drawPlayersTexts()
         self.drawCircles()
-
         self.drawAllPlayersInLobby()
 
-        pygame.display.update()
+    def showLobby(self):
+        self.showLobbyBackground()
+        mousePos = pygame.mouse.get_pos()
+
+        backButton = Button(pos=(CENTER_X_POS, 640), textInput="BACK", font=getFont(50),
```

Comment:
We can set screen width, height and coordinates as constants

Reply:
Done